lib/Db/Feed.php

```diff
@@ -648,6 +660,15 @@ public function setUserId(string $userId): Feed
         return $this;
     }
 
+    /**
+     * @param int $nextUpdateTime
+     */
+    public function setNextUpdateTime(int $nextUpdateTime): void
```

Comment:
This should be nullable, since you mentioned feeds can be without it too.
```suggestion
    public function setNextUpdateTime(?int $nextUpdateTime): void
```